src/Card/Card.scss

```diff
@@ -248,3 +248,23 @@
     flex-direction: row;
   }
 }
+
+@media (max-width: map-get($grid-breakpoints, "sm")) {
+  .pgn__card-footer {
+    .btn {
+      width: 100%;
+      margin: .625rem 0 0;
+      margin-inline-end: 0 !important;
```

Comment:
Is `!important` necessary here? We should generally avoid using that in our styles since it's hard to override for consumers

Reply:
Yes, you are right. 
This was a working variant, I forget to delete it.
Thanks